src/trusted/service_runtime/nacl_syscall_common.c

```diff
@@ -3628,6 +3863,10 @@ int32_t NaClSysWaitpid(struct NaClAppThread *natp,
   NaClLog(1, "[NaClSysWaitpid] options = %d \n", options);
   NaClLog(1, "[NaClSysWaitpid] ret = %d \n", ret);
 
+  #ifdef TRACING
+  NaClStraceWaitpid(nap->cage_id, pid, stat_loc, options, ret);
```

Comment:
is this valid?

Reply:
stat loc is 32 bit, you want statlocptr i believe?

Reply:
if you look at the syscall function, you see that `stat_loc` is a pointer passed into the syscall by the user, which is a user address, and there is a `sysaddr` variable declared to be the translated system address. I think using the sysaddr makes more sense here

Reply:
there is no lind_* call happening inside it so i was confused